features/step_definitions/purchase_steps.rb

```diff
@@ -22,9 +22,9 @@
 
 Then(/^the book is added to my cart$/) do
   @cart = @user.cart
-  expect(@cart.order_items).to_not eq(nil)
-  last_order_item = @cart.order_items.last
-  expect(last_order_item.book).to eq(@book)
+  expect(@cart.line_items).to_not eq(nil)
+  last_line_item = @cart.line_items.last
+  expect(last_line_item.book).to eq(@book)
```

Comment:
Oh, actually, I see you're doing that on line 30. Maybe this test is just extraneous.